.github/workflows/check-md-links.yml

```diff
@@ -0,0 +1,26 @@
+name: Markdown Links Check
+# runs every monday at 9 am
+on:
+  schedule:
+    - cron: "0 9 * * 1"
+
+jobs:
+  check-links:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - uses: gaurav-nelson/github-action-markdown-link-check@v1
+        # checks all markdown files from /docs including all subfolders
+        with:
+          use-quiet-mode: 'yes'
+          use-verbose-mode: 'yes'
+          config-file: 'markdown.links.config.json'
```

Comment:
can this config-file be anywhere? If so, can we move it under `.github/workflows/` to keep it out of the root directory?

Reply:
sure, it's done!